modules/electrophysiology_browser/test/electrophysiologyBrowserTest.php

```diff
@@ -268,7 +268,8 @@ function testEEGBrowserWithSitePermissions()
      */
     function testFilters()
     {
-        $this->safeGet($this->url . "/electrophysiology_browser/");
+        $this->markTestSkipped('Must be updated with annotation additions to pass');
```

Comment:
```suggestion
        $this->markTestSkipped('Must be updated with annotations to pass');
```
is this suggestion here more accurate?